requirements-base.txt

```diff
@@ -56,7 +56,7 @@ rfc3986-validator==0.1.1
 # [end] jsonschema format validators
 sentry-arroyo==0.2.0
 sentry-relay==0.8.12
-sentry-sdk>=1.4.3,<1.6.0
+sentry-sdk==1.6.0
```

Comment:
we're in the process of freezing everything -- more context in https://github.com/getsentry/sentry/pull/35919

Reply:
why not just do this though? we already pin all other deps

Reply:
I'm not opposed to this -- I was just commenting so there was additional context from the previous discussion of this library

Reply:
Ah thanks for the context, I was looking since I assumed 1.6 came up but couldn't find it after a cursory glance, but I agree with Markus, we can just get this in for now while also working on long term.